### PR TITLE
Add bootc status

### PR DESCRIPTION
**Pull request by JoySnow**
[RHINENG-7612](https://issues.redhat.com/browse/RHINENG-7612)

Related inventory-schema adding at https://github.com/RedHatInsights/inventory-schemas/pull/123

### CODE REVIEW

**Comment by JoySnow** [comment]
The local test looks good with: 
```
--------------
Rules Executed
--------------
[META] src.puptoo.process.profile.system_profile
------------------------------------------------

    bootc_status           : {'booted': {'image': '192.168.124.1:5000/bootc-insights:latest',
                                         'image_digest': 'sha256:806d77394f96e47cf99b1233561ce970c94521244a2d8f2affa12c3261961223'},
                              'rollback': {'image': 'quay.io/centos-boot/fedora-boot-cloud:eln',
                                           'image_digest': 'sha256:92e476435ced1c148350c660b09c744717defbd300a15d33deda5b50ad6b21a0'}}
```

File: src/puptoo/process/profile.py [hunk 1]
```diff
@@ -644,6 +647,22 @@ def system_profile(
     if gcp_network_interfaces:
         profile["public_ipv4_addresses"] = _remove_empty_string(gcp_network_interfaces.public_ips)
 
+    if bootc_status:
+        try:
+            profile["bootc_status"] = {}
+            # Another filed "cachedUpdate" could be added if required later
+            status = bootc_status.get('status', {})
+            for bootc_key in ["booted", "staged", "rollback"]:
```
Comment (r14chandra) [inline src/puptoo/process/profile.py:655]: As per the schema under inventory-repo, we have not considered [cachedUpdate](https://github.com/RedHatInsights/inventory-schemas/blob/master/schemas/system_profile/v1.yaml#L678C11-L678C23). Is there any reason?

Comment (JoySnow) [reply src/puptoo/process/profile.py:655]: Hi @r14chandra , yes, for the "cachedUpdate",  we are skipping it at this stage, referring to the `Description` part  of this [RHINENG-7612](https://issues.redhat.com/browse/RHINENG-7612) by our initiator's:

> There is a 'cachedUpdate' field that I am still gathering info on. It is OK to skip populating that for now and we can revisit it later.

Comment (r14chandra) [reply src/puptoo/process/profile.py:655]: Thanks for confirming @JoySnow.

**Review by r14chandra — APPROVED** [review]
Thanks @JoySnow for working on it! The PR looks to me. 👍🏻

**Comment by JoySnow** [comment]
Thanks a lot for the review, @r14chandra .

**Review by xiangce — APPROVED** [review]
Looks good to me too.

**Pull request merged**